k8s-operator/apis/v1alpha1/types_connector.go

```diff
@@ -156,6 +156,13 @@ type ConnectorStatus struct {
 	// IsExitNode is set to true if the Connector acts as an exit node.
 	// +optional
 	IsExitNode bool `json:"isExitNode"`
+	// TailnetIPs is the set of tailnet IP addresses (both IPv4 and IPv6)
+	// assigned to the Connector node(s).
+	// +optional
+	TailnetIPs []string `json:"tailnetIPs,omitempty"`
+	// Hostname is the fully qualified domain name of the Connector node(s).
```

Comment:
Nit: currently one one node is possible (and if we allow multiple, each will have its own hostname)